.gitignore

```diff
@@ -40,4 +40,5 @@ gaffer-as-a-service/gaas-rest/.classpath
 
 gaffer-as-a-service/gaffer-controller/.classpath
 
-gaas-ui/package-lock.json
+gaffer-as-a-service/gaas-ui/package-lock.json
+/gaffer-as-a-service/gaas-ui/node_modules/
```

Comment:
This relies on https://github.com/gchq/gaffer-experimental/pull/295

Reply:
Changed as per the review comments